### PR TITLE
add(postgresql): expose pgbouncer stats to datadog

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/BF-2449

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by 0xlianhu — COMMENTED** [review]
It looks good to me.

File: docs/products/postgresql/howto/monitor-database-with-datadog.md [hunk 1]
```diff
@@ -84,8 +81,8 @@ Monitoring for your service.
 
 ## Related pages
 
--   Learn more about
-    [Datadog and Aiven](/docs/integrations/datadog).
--   Learn more about [Datadog Deep Database
-    Monitoring](https://www.datadoghq.com/product/database-monitoring/)
-    from their product page.
+- Learn more about [Datadog and Aiven](/docs/integrations/datadog).
+- [Expose PgBouncer statistics to Datadog](/docs/products/postgresql/howto/pgbouncer-stats-datadog).
```
Comment (divdby0) [inline docs/products/postgresql/howto/monitor-database-with-datadog.md:85]: I recommend "PgBouncer monitoring with Datadog" analogous to our "Database monitoring with Datadog".

File: docs/products/postgresql/howto/monitor-database-with-datadog.md [hunk 2]
```diff
@@ -84,8 +81,8 @@ Monitoring for your service.
...
+- Learn more about
+  [Datadog Deep Database Monitoring](https://www.datadoghq.com/product/database-monitoring/)
```
Comment (divdby0) [inline docs/products/postgresql/howto/monitor-database-with-datadog.md:87]: This is a minor thing but Datadog dropped the term "Deep" from their offering name (it is only visible in the main page headline but not referred to in any other place) so I recommend changing this to "Database monitoring with Datadog" to keep in line with the title of our own doc (https://aiven.io/docs/products/postgresql/howto/monitor-database-with-datadog).

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+title: Expose PgBouncer statistics to Datadog
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:2]: I recommend "PgBouncer monitoring with Datadog"

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+sidebar_label: PgBouncer stats in Datadog
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:3]: I recommend "PgBouncer monitoring with Datadog".

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+Connect PgBouncer with Datadog to display [PgBouncer statistics](/docs/products/postgresql/howto/pgbouncer-stats) on the Datadog platform.
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:6]: I recommend "Integrate/Connect PgBouncer with Datadog to track connection pool metrics, and monitor application traffic on the Datadog platform".

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+## Prerequisites
+
+- Your service plan is Startup or higher.
+- [Datadog metrics integration](/docs/integrations/datadog/datadog-metrics) is
+  enabled on your service.
+- You applied outstanding service maintenance updates affecting the Datadog metrics
+  integration.
+
+## Expose PgBouncer statistics
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:16]: I recommend "Enable monitoring" to drop the term "statistics" as well as to keep it consistent with "Enable monitoring" in [Database monitoring with Datadog :: Enable monitoring](https://aiven.io/docs/products/postgresql/howto/monitor-database-with-datadog#enable-monitoring)

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+You can expose PgBouncer statistics for your Aiven for PostgreSQL service both if the
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:18]: I recommend to drop the terms "expose" and "statistics" in favor of "metrics" and "monitoring". I recommend the opening similar/consistent to the one in Database monitoring: "To use Datadog PgBouncer Monitoring with your Aiven for PostgreSQL® services, ...".

I am also not clear about the "regardless if this service already has a Datadog Metrics integration or not." part. Then, this begs the question of why Datadog metrics integration is a prerequise in the first place.
@0xlianhu could you please clarify?

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+service already has a Datadog Metrics integration and if it doesn't.
+
+### Expose PgBouncer stats if the integration exists
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:21]: I suggest to drop the "expose" wording or this subsection title altogether analogous to [Database monitoring with Datadog :: Enable monitoring](https://aiven.io/docs/products/postgresql/howto/monitor-database-with-datadog#enable-monitoring).

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+1. Obtain the SERVICE_INTEGRATION_ID of the Datadog Metrics integration for your Aiven for
+   PostgreSQL service by running the
+   [avn service integration-list](/docs/tools/cli/service/integration#avn_service_integration_list)
+   command:
+
+   ```bash
+   avn service integration-list --project PROJECT_NAME SERVICE_NAME
+   ```
+
+1. To enable PgBouncer statistics in Datadog, set up the ``datadog_pgbouncer_enabled``
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:32]: I recommend dropping "statistic" or using something like "... PgBouncer monitoring ...".

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+   parameter to ``true``:
+
+   ```bash
+   avn service integration-update --project PROJECT_NAME --user-config '{"datadog_pgbouncer_enabled": true}' SERVICE_INTEGRATION_ID
+   ```
+
+   Replace SERVICE_INTEGRATION_ID with the service integration identifier acquired in the
+   preceding step.
+
+### Create the integration and expose PgBouncer stats
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:42]: I recommend dropping the second part with the term "expose".
"Create the integration"

File: docs/products/postgresql/howto/pgbouncer-stats-datadog.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+With no Datadog Metrics integration in place,
+[create the integration for your service](/docs/tools/cli/service/integration#avn_service_integration_create)
+and enable PgBouncer statistics in Datadog by running:
```
Comment (divdby0) [inline docs/products/postgresql/howto/pgbouncer-stats-datadog.md:46]: I recommend replacing "statistics" with "monitoring".

File: sidebars.ts [hunk 12]
```diff
@@ -1932,6 +1932,7 @@ const sidebars: SidebarsConfig = {
                     'products/postgresql/howto/claim-public-schema-ownership',
                     'products/postgresql/howto/manage-pool',
                     'products/postgresql/howto/pgbouncer-stats',
+                    'products/postgresql/howto/pgbouncer-stats-datadog',
```
Comment (divdby0) [inline sidebars.ts:1935]: I recommend the URI "pgbouncer-datadog" or "monitor-pgbouncer-with-datadog".

File: docs/integrations/datadog/datadog-metrics.md [hunk 13]
```diff
@@ -59,6 +59,7 @@ metric tracking in Datadog, follow these steps:
 ## Related pages
 
 - Learn more about [Datadog and Aiven](/docs/integrations/datadog).
+- [Expose PgBouncer statistics to Datadog](/docs/products/postgresql/howto/pgbouncer-stats-datadog).
```
Comment (divdby0) [inline docs/integrations/datadog/datadog-metrics.md:62]: I recommend "[PgBouncer monitoring with Datadog](/docs/products/postgresql/howto/monitor-pgbouncer-with-datadog)".

File: docs/products/postgresql/howto.md [hunk 14]
```diff
@@ -41,6 +41,7 @@ Database administration
 -   [Claim public schema ownership](/docs/products/postgresql/howto/claim-public-schema-ownership)
 -   [Manage connection pooling](/docs/products/postgresql/howto/manage-pool)
 -   [Access PgBouncer statistics](/docs/products/postgresql/howto/pgbouncer-stats)
+-   [PgBouncer statistics in Datadog](/docs/products/postgresql/howto/pgbouncer-stats-datadog)
```
Comment (divdby0) [inline docs/products/postgresql/howto.md:44]: I recommend "[PgBouncer monitoring with Datadog](/docs/products/postgresql/howto/monitor-pgbouncer-with-datadog)".

**Review by divdby0 — APPROVED** [review]

File: docs/products/postgresql/howto/monitor-pgbouncer-with-datadog.md [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+You can enable monitoring PgBouncer metrics for your Aiven for PostgreSQL service both
+[if the service already has a Datadog Metrics integration](#enable-monitoring-if-the-integration-exists)
+and [if it doesn't](#create-the-integration-and-enable-monitoring).
+
+### Enable monitoring for an integrated service{#enable-monitoring-if-the-integration-exists}
+
+Enable monitoring PgBouncer metrics for an Aiven for PostgreSQL service that already
+has a Datadog Metrics integration:
+
+1. Obtain the SERVICE_INTEGRATION_ID of the Datadog Metrics integration for your Aiven for
```
Comment (ArthurFlag) [inline docs/products/postgresql/howto/monitor-pgbouncer-with-datadog.md:25]: Can we put variable names as inline code to help with scanability (applies to the whole file)?

```suggestion
1. Obtain the `SERVICE_INTEGRATION_ID` of the Datadog Metrics integration for your Aiven for
```

File: docs/products/postgresql/howto/monitor-pgbouncer-with-datadog.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+   [avn service integration-list](/docs/tools/cli/service/integration#avn_service_integration_list)
+   command:
+
+   ```bash
+   avn service integration-list SERVICE_NAME \
+      --project PROJECT_NAME
+   ```
+
+1. To enable PgBouncer monitoring in Datadog, set up the ``datadog_pgbouncer_enabled``
+   parameter to ``true``:
```
Comment (ArthurFlag) [inline docs/products/postgresql/howto/monitor-pgbouncer-with-datadog.md:36]: Applies to the whole file: only one backtick for inline code in MD :) 

```suggestion
1. To enable PgBouncer monitoring in Datadog, set up the `datadog_pgbouncer_enabled`
   parameter to `true`:
```

**Review by ArthurFlag — CHANGES_REQUESTED** [review]
minor md syntax comments

**Pull request merged**